examples/blog/templates/feed.jade

```diff
@@ -21,6 +21,7 @@ rss(version='2.0',
         pubDate= article.rfc822date
         guid(isPermaLink='true')= permalink
         - var author = contents.authors[article.metadata.author + '.json'];
+        - if (!author) {author = { metadata: {name: null}} };
```

Comment:
That breaks previous behaviour where you can use author just as a string if no author.json is found. Better to do it the other way around and only use author.json if it is present, otherwise just use the string.

Reply:
Ah yes, I see your point there!